public/app/utilities.js

```diff
@@ -135,8 +136,21 @@ const clean = async dir => {
   return promisify(rimraf)(dir);
 };
 
+/**
+ * Ensure directories for given filepath exist
+ */
+function ensureDirectoryExistence(filePath) {
```

Comment:
Maybe we should use `fs-extra` instead of creating this function? Probably has some in-built safety.

```
const fs = require('fs-extra');
fs.ensureDirSync(...);
```